stores/categories.ts

```diff
@@ -1,28 +1,41 @@
-import { defineStore } from 'pinia'
+import { defineStore } from 'pinia';
 
-interface State {
-  categories: Category[]
-}
+export const useCategoryStore = defineStore('categories', () => {
+  const categories = ref([]);
```

Comment:
Waar komt die ref functie vandaan? Die wordt niet geïmporteerd.

Reply:
Auto import van Nuxt

Reply:
> Verder lees ik dat het gebruik van ref buiten setup wordt afgeraden: https://v3.nuxtjs.org/getting-started/state-management/#best-practices

Gelukkig doe ik dat ook niet ;)
https://pinia.vuejs.org/core-concepts/#setup-stores

Dit is de setup notatie van een pinia store. Ik moet ref gebruiken om er een state van te maken.